server/application-server/src/main/resources/application-prod.yml

```diff
@@ -28,3 +30,9 @@ monitoring:
 
 github:
   authToken: ${GITHUB_AUTH_TOKEN}
+slack:
```

Comment:
I think we should slowly move configuration under `hephaestus`, i.e. `hephaestus.notification.slack`. Or we keep `slack` at the top level and move some parts into `hephaestus.notification`, i.e. `hephaestus.notification.leaderboard.weeky.enabled`

Reply:
Since we can easily use the Slack-bot for other use cases as well though, moving the whole Slack-section under `notification` wouldn't be fair.
All notification-specific settings are now under `hephaestus.notification` as suggested (ce267df).